test/ffmpeg-vaapi/vpp/transpose.py

```diff
@@ -32,6 +32,9 @@ def test(self, case, degrees, method):
       method    = method,
     )
 
+    if get_media()._get_os() == 'wsl':
+      slash.skip_test("vaon12 driver does not support transpose")
```

Comment:
"wsl" does not imply "vaon12".  If vaon12 does not support this, then it should be removed from the "vaon12" caps files, instead.

Reply:
Remove transpose will also remove the mirroring and rotation support. Is there any way to avoid that?